src/App.js

```diff
@@ -65,16 +65,6 @@ function App() {
     }
   };
 
-  let but;
-  but = (
-    <input
-      className={`input ${dark ? "dark_input" : "light_input"}`}
-      placeholder="Write Your Message"
-      value={input}
-      onChange={(event) => setInput(event.target.value)}
-    />
-  );
-
```

Comment:
You deleted this ! Is the app still working ?? Do send the gif or video for the same.